src/app/catalog/product/components/product-master/product-master.component.ts

```diff
@@ -11,7 +11,9 @@ import {
   Component,
   OnInit,
 } from '@angular/core';
+import { faComment } from '@fortawesome/free-solid-svg-icons';
 import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
```

Comment:
In this file, you're importing the `FontAwesomeModule` while in `product-detail.component.ts` you're importing the `FaIconComponent`. 
In both cases, in the end you're using the `faComment` icons. Why is there a difference in the setup/import?